module/tooltips.mjs

```diff
@@ -114,14 +114,28 @@ export default class Tooltips5e {
   /* -------------------------------------------- */
 
   /**
-   * Handle hovering a passive check link to display results for primary party.
-   * @param {string} skill  Passive skill key.
-   * @param {number} [dc]   DC against which to compare party values.
+   * Handle hovering a passive skill or ability check link to display results for primary party.
+   * Either skill or ability (or both) must be provided.
+   * @param {string} [skill]     Passive skill key. If undefined, this will be a passive ability check.
+   * @param {string} [ability]   Passive ability key. If undefined, the skill's default ability is used.
+   * @param {number} [dc]        DC against which to compare party values.
    * @protected
    */
-  async _onHoverPassive(skill, dc) {
-    const config = CONFIG.DND5E.skills[skill];
-    const label = game.i18n.format("DND5E.SkillPassiveHint", { skill: config.label });
+  async _onHoverPassive(skill, ability, dc) {
+    const skillConfig = CONFIG.DND5E.skills[skill];
+    const abilityConfig = CONFIG.DND5E.abilities[ability ?? skillConfig.ability];
+
+    let label;
+    if ( skillConfig ) {
+      label = game.i18n.format("EDITOR.DND5E.Inline.SpecificCheck", { type: skillConfig.label, ability: abilityConfig.label });
+    } else {
+      // If no skill was provided, we're doing a passive ability check.
+      // This isn't technically a thing in the rules, but we can support it anyway if people want to use it.
+      label = abilityConfig.label;
+    }
+
+    label = game.i18n.format("EDITOR.DND5E.Inline.DCPassiveShort", { dc, check: label });
```

Comment:
Not a big fan of how this appears, with the lowercase "passive". I'd say we add a new localization key to go along with `DND5E.SkillPassiveHint` that also has the ability.

`"Passive Wisdom"`
`"Passive Wisdom (Perception)"`

Reply:
Yeah, I considered that as an option as well. I figured it might be useful to have the dc in the tooltip title so I used the existing template, but I'll add a new SkillPassiveSpecificHint instead